packages/grafana-data/src/themes/createTypography.ts

```diff
@@ -65,11 +67,13 @@ export interface ThemeTypographyInput {
 
 const defaultFontFamily = '"Roboto", "Helvetica", "Arial", sans-serif';
 const defaultFontFamilyMonospace = "'Roboto Mono', monospace";
+const defaultFontFamilyMonospaceFusebit = "'Courier Prime', monospace";
```

Comment:
If there's no optionality here (i.e. there's no way to specify this at runtime - in which case, why is it called `default`? Is there a configuration-file change here that could be made instead?) then let's just change the `defaultFontFamilyMonospace` because that will produce a smaller diff.